localization/ch.xml

```diff
@@ -1,57 +1,57 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <localizationPack name="Switzerland" version="1.0">
 	<currencies>
-		<currency name="Franc" iso_code="CHF" iso_code_num="756" sign="CHF" blank="1" conversion_rate="1.34183" format="5" decimals="1" />
+		<currency name="Swiss Franc" iso_code="CHF" iso_code_num="756" sign="CHF" blank="1" conversion_rate="1.34183" format="5" decimals="1" />
 	</currencies>
 	<languages>
 		<language iso_code="de" />
 		<language iso_code="fr" />
 		<language iso_code="it" />
 	</languages>
 	<taxes>
-		<tax id="1" name="TVA CH 7.7%" rate="7.7" />
-		<tax id="2" name="TVA CH 3.7%" rate="3.7" />
-		<tax id="3" name="TVA CH 2.5%" rate="2.5" />
+		<tax id="1" name="VAT CH 7.7%" rate="7.7" />
+		<tax id="2" name="VAT CH 2.5%" rate="2.5" />
+		<tax id="3" name="VAT CH 3.7%" rate="3.7" />
```

Comment:
Hi @swiss-guy, thanks for your contribution ! I'm just wondering if it's necessary to change the tax id? Why not just keeping the original id with your naming changes?
```suggestion
		<tax id="1" name="VAT CH 7.7%" rate="7.7" />
		<tax id="2" name="VAT CH 3.7%" rate="3.7" />
		<tax id="3" name="VAT CH 2.5%" rate="2.5" />
```

Reply:
Hi there, same question: why would you change the order?